server/config/feature_flag_configs/autopush.json

```diff
@@ -41,6 +41,18 @@
     "owner": "nick-next",
     "description": "Feature flag to show the complete metadata modal. If not set, the older modal will be used as a fallback."
   },
+  {
+    "name": "page_overview_experiment",
+    "enabled": false,
+    "owner": "javiervazquez",
+    "description": "Enables the page overview generated from query and relevant stat vars as an experiment."
```

Comment:
It would be helpful to document what you mean by "experiment" here. 

For example, project 1's description would include "enables a 20% chance of the feature being displayed. Once the feature has been displayed, it will continue to be displayed for the entirety of the user's session." 

If you're not sure of this experiment's mechanics yet, then feel free to submit as is, but consider coming back and making this more descriptive once it has been decided.